_data/projects/DSA.yml

```diff
@@ -0,0 +1,9 @@
+name: DSA
+desc: Data structure and Algorithm (DSA) contributions
+site: https://github.com/MakeContributions/DSA
+tags:
+- dsa
+- algorithm
+upforgrabs:
+  name: help wanted
+  link: https://github.com/MakeContributions/DSA/labels/help%20wanted
```

Comment:
@ming-tsai Please mark existing issue on your repo with this label or open new issues with this label.

Reply:
Done check it out